common/common_helper.sh

```diff
@@ -294,3 +294,23 @@ function check_minikube() {
 	fi
 }
 
+###########################################
+#   Deploy TFB Benchmarks - multiple import
+###########################################
+function create_namespace_and_install_benchmarks() {
+	echo
+	echo "#######################################"
+	echo "Creating new namespace: test-multiple-import"
+  	kubectl create namespace test-multiple-import
+	echo "#######################################"
+	pushd benchmarks >/dev/null
```

Comment:
@shreyabiradar07 section of code which installs benchmarks looks duplicated. Would be good if you can use the existing benchmarks_install function and only have create_namespace as a different function.
You can modify the existing benchmarks_install function to use the namespace.

Reply:
@kusumachalasani removed the duplicate code to install benchmarks, adding custom namespace support and also updated the manifests to `manifests/default_manifests`